provision/aws/aws_add_secret_to_openshift_env_file.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+source ./.env
+
+KEYCLOAK_MASTER_PASSWORD_SECRET_NAME=${KEYCLOAK_MASTER_PASSWORD_SECRET_NAME:-"keycloak-master-password"}
+# Force eu-central-1 region for secrets manager so we all work with the same secret
+REGION="eu-central-1"
+KEYCLOAK_MASTER_PASSWORD=$(aws secretsmanager get-secret-value --region $REGION --secret-id "$KEYCLOAK_MASTER_PASSWORD_SECRET_NAME" --query SecretString --output text --no-cli-pager)
+
+echo "KC_ADMIN_PASSWORD='$KEYCLOAK_MASTER_PASSWORD'" >> ../openshift/.env
+echo "Password added to .env file."
```

Comment:
It would be nice to mention this in the docs with something like 

```

Optionally we can use `aws_add_secret_to_openshift_env_file.sh` script to set up the openshift `.env` variable `KC_ADMIN_PASSWORD` with the newly generated admin password, which will then be re-used for Keycloak Admin and Grafana Admin secure deployments with dynamic rotatable admin password.
```